cupy/linalg/solve.py

```diff
@@ -156,15 +156,49 @@ def inv(a):
 
     .. seealso:: :func:`numpy.linalg.inv`
     '''
-    if not cuda.cusolver_enabled:
-        raise RuntimeError('Current cupy only supports cusolver in CUDA 8.0')
 
     util._assert_cupy_array(a)
     util._assert_rank2(a)
     util._assert_nd_squareness(a)
 
-    b = cupy.eye(len(a), dtype=a.dtype)
-    return solve(a, b)
+    if not cuda.cusolver_enabled:
+        raise RuntimeError('Current cupy only supports cusolver in CUDA 8.0')
```

Comment:
Only in CUDA 8.0? I think it's better to say `CUDA>=8.0`

Reply:
It is better to fix all these message. I'll do it in another PR.

Reply:
OK